packages/components/nodes/memory/ZepMemory/ZepMemory.ts

```diff
@@ -196,6 +197,28 @@ class ZepMemoryExtended extends ZepMemory {
         }
         return super.clear()
     }
+
+    async getChatMessages(overrideSessionId = '', returnBaseMessages = false): Promise<IMessage[] | BaseMessage[]> {
```

Comment:
Should this class implement MemoryMethods?

Reply:
implemented `MemoryMethods`